tests/sqlparser_clickhouse.rs

```diff
@@ -55,7 +55,7 @@ fn parse_map_access_expr() {
                         "indexOf",
                         [
                             Expr::Identifier(Ident::new("string_names")),
-                            Expr::Value(Value::SingleQuotedString("endpoint".to_string()))
+                            Expr::value(Value::SingleQuotedString("endpoint".to_string()))
```

Comment:
By adding the `Expr::value` method, the changes to update the tests became a find/replace exercise